api/src/main/java/ca/bc/gov/educ/eas/api/service/v1/PENRecordService.java

```diff
@@ -0,0 +1,24 @@
+package ca.bc.gov.educ.eas.api.service.v1;
+
+import ca.bc.gov.educ.eas.api.rest.RestUtils;
+import ca.bc.gov.educ.eas.api.struct.v1.StudentMerge;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class PENRecordService {
+
+    private final RestUtils restUtils;
+
+    @Autowired
+    public PENRecordService(RestUtils restUtils){
+        this.restUtils = restUtils;
+    }
+
+    public List<StudentMerge> getMergedStudentsForDateRange(String createDateStart, String createDateEnd){
+        return restUtils.getMergedStudentsForDateRange(UUID.randomUUID(), createDateStart, createDateEnd);
```

Comment:
Do we need this randomID ?

Reply:
as far as I know, we do - it is only a transaction id that will be shared between the student api and assessments. basically grabbed this directly from how we are retrieving merges in sdc

Reply:
@eckermania - It's only useful if we log in on this side.